Snakefile

```diff
@@ -157,6 +157,7 @@ rule clean_osm_data:
         generators="data/clean/africa_all_generators.geojson",
         generators_csv="data/clean/africa_all_generators.csv",
         lines="data/clean/africa_all_lines.geojson",
+        links="data/clean/africa_all_links.geojson",
```

Comment:
Thinking again, maybe we can use the keyword "converters" instead of links, this complies also with pypsa-eur, given the intended content of these outputs